README.md

```diff
@@ -23,6 +23,7 @@ However, you can also build it as a standalone project like so:
 git clone https://github.com/vshaxe/haxe-language-server
 cd haxe-language-server
 npm install
+npx lix download
```

Comment:
shouldn't `postinstall` in `package.json` take care of that when doing `npm install`?

Reply:
Yep, this shouldn't be needed.

Reply:
OK, I removed the line again. I had to manually download the dependencies before since lix was breaking because of the whitespace issue.